Initium-Odp/war/odp/ajax_group.jsp

```diff
@@ -258,7 +258,10 @@
 								+ ")'>Demote from admin</a>");
 					}
 				}
-				if (groupStatus.equals("Kicked") == false)
+				if (groupStatus.equals("Kicked") == false 
+						&& groupStatus.equals("Admin") == false 
+						|| (Key) group.getProperty("creatorKey")).getId() == 
```

Comment:
This file's no longer being used so changes to it should be discarded. Made inferior by ajax_group.jsp in odppages